lib/logstash/devutils/rspec/shared_examples.rb

```diff
@@ -0,0 +1,19 @@
+require 'rspec/wait'
+
+RSpec.shared_examples "an interruptible input plugin" do
+  describe "stop" do
```

Comment:
I tend to use `#foo` syntax in descriptions ot indicate that it's a method, but i'm not strongly opinionated here.